content/blog/2019-10-02-dev-guide-update-fall19.md

```diff
@@ -0,0 +1,94 @@
+
+We've added a [book release guide](https://github.com/ropensci/dev_guide/issues/152) - a guide for shepherding a new version of the developer guide - which culminates in this blog post!
+
+### Guide tweaks
+
+The developer guide now lives at <https://devguide.ropensci.org/>!
+
+We fixed formatting of Appendices B-D in the pdf version of the book (see PDF button in the navbar) with [an issue](https://github.com/ropensci/dev_guide/issues/179) by [Indrajeet Patil](https://github.com/IndrajeetPatil).
+
+For book contributors, we've update [instructions in the README](https://github.com/ropensci/dev_guide#notes-for-associate-editors) for rendering the developer guide locally (<https://github.com/ropensci/dev_guide/issues/192>).
+
+[Hugo Gruson][hugog] helped us with a number of things: added a note for package maintainers about the [importance of GitHub recognizing your repository as the language R](https://github.com/ropensci/dev_guide/pull/172), and helped us [update links to `https` from `http`](https://github.com/ropensci/dev_guide/pull/167).
+
+
+
+### Updates to our policies and guidance
+
+#### Scope
+
+We've introduced an important change for anyone thinking of submitting a package. The former "reproducibility" category has been split up into four new categories (with "reproducibility" itself removed). They are:
+
+- data validation and testing
+- workflow automation
+- version control
+- citation management and bibliometrics
+
+For discussion and changes see [ropensci/software-review-meta#81](https://github.com/ropensci/software-review-meta/issues/81) and [ropensci/dev_guide#184](https://github.com/ropensci/dev_guide/pull/184).
+
+#### Documentation
+
+rOpenSci's [Jeroen Ooms](https://ropensci.org/authors/jeroen-ooms/) has built out a new way for rOpenSci packages to improve documentation for their packages. Many of you have likely been building package documentation websites with [pkgdown][]. There were ways to automate the building of these documentation pages - but nothing as easy as we have now. Docs for each rOpenSci package are now built automatically and can be found at `https://docs.ropensci.org/{package}`, where `{package}` is the package name, e.g., the page for the [bowerbird][] package is <https://docs.ropensci.org/bowerbird/>.
+
+We've changed our guidelines regarding README's and vignettes. If you have chunks shared between the README and any vignettes, then [we recommend using Rmarkdown chunks](https://devguide.ropensci.org/building.html#documentation). Related issues: [ropensci/dev_guide#159](https://github.com/ropensci/dev_guide/issues/159) and [ropensci/dev_guide#161](https://github.com/ropensci/dev_guide/issues/161).
+
+Last, we've removed our recommendation about using the `roxygen2` development version because the latest stable version has what is needed regarding tags and the `rev` contributor role. (thanks [Hugo Gruson][hugog] - [ropensci/dev_guide#165](https://github.com/ropensci/dev_guide/issues/165))
```

Comment:
```suggestion
Last, we've removed our recommendation about using the `roxygen2` development version because the latest stable version has what is needed regarding tags and the `rev` contributor role. (thanks again [Hugo Gruson][hugog] - [ropensci/dev_guide#165](https://github.com/ropensci/dev_guide/issues/165))
```